Hotel-Management.py

```diff
@@ -39,73 +18,51 @@ def menu():
         print(f"{num}: {option.get('title')}")
     print()
 
-    options.get( int(input("Enter your choice(1-6): ")) ).get("method")()
+    options.get(int(input("Enter your choice(1-6): "))).get("method")()
 
 
 def add():
-
-    Name1 = input("\nEnter your first name: \n")
-    Name2 = input("\nEnter your last name: \n")
-    Phone_Num = input("\nEnter your phone number(without +91): \n")
+    Name1 = input("\nEnter your first name: ")
+    Name2 = input("\nEnter your last name: ")
+    Phone_Num = input("\nEnter your phone number (without +91): ")
 
     print("These are the rooms that are currently available")
     print("1-Normal (500/Day)")
     print("2-Deluxe (1000/Day)")
     print("3-Super Deluxe (1500/Day)")
     print("4-Premium Deluxe (2000/Day)")
 
-    Room_Type = int(input("\nWhich type you want(1-4): \n"))
-
-    match Room_Type:
-        case 1:
-            x = 500
-            Room_Type = "Normal"
-        case 2:
-            x = 1000
-            Room_Type = "Deluxe"
-        case 3:
-            x = 1500
-            Room_Type = "Super Deluxe"
-        case 4:
-            x = 2000
-            Room_Type = "Premium"
-
-    Days = int(input("How many days you will stay: "))
-    Money = x * Days
-    Money = str(Money)
-    print("")
+    Room_Type = int(input("\nWhich type you want (1-4): "))
 
-    print("You have to pay ", (Money))
-    print("")
+    room_types = {
+        1: ("Normal", 500),
+        2: ("Deluxe", 1000),
+        3: ("Super Deluxe", 1500),
+        4: ("Premium Deluxe", 2000),
+    }
 
-    Payment = input("Mode of payment(Card/Cash/Online): ").capitalize()
-    if Payment == "Card":
-        print("Payment with card")
-    elif Payment == "Cash":
-        print("Payment with cash")
-    elif Payment == "Online":
-        print("Online payment")
-    print("")
+    Room_Type, x = room_types.get(Room_Type, ("Unknown", 0))
+
+    Days = int(input("How many days will you stay: "))
+    Money = x * Days
+
+    print(f"\nYou have to pay {Money:.2f}")
+    Payment = input("Mode of payment (Card/Cash/Online): ").capitalize()
+    print()
 
-    with open("Management.txt", "r") as File:
-        string = File.read()
-        string = string.replace("'", '"')
-        dictionary = json.loads(string)
+    with open("Management.txt", "r") as file:
+        dictionary = json.load(file)
 
-    if len(dictionary.get("Room")) == 0:
+    if not dictionary["Room"]:
```

Comment:
Its not recommended in this case to directly fetch "Room".. instead its a good practice to use `.get()` because if the value Room is not available in the dictionary the script will error and halt however `.get()` is going to return `None` type so the script can continue forward without errors